### PR TITLE
Toy Tesla

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR
A toy tesla! That's shocking!
it does 2 damage because for some reason 1 or less damage doesn't cause a micro stun
## Why / Balance

because there's a toy singularity

## Media

https://github.com/space-wizards/space-station-14/assets/96445749/344c613a-30b2-4f1f-9d1b-d0150caa6569

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
no cl, it needs to be added to the secrets

### CODE REVIEW

**Comment by K-Dynamic** [comment]
I hope we can emag it for something fun

**Comment by UbaserB** [comment]
based

**Comment by Cyber-Finn** [comment]
lol so I defs think this is cool af, and was hoping for this to be added (So TYSM!) 🥳 
But I have some questions regarding the impact it could have on gameplay - if tiders, for example, exploited it.

Here are my questions:
1. How many of these Teddy Teslas can be found on each map?
2. How accessible are they (how easy to find)?
3. How likely is it that a tider/antag could use this to stun sec, cuff them and steal gear, etc.?

I ask question 3 specifically because it took you 2 (I think) stuns to be at "okay" health, I'm assuming another 2 would bring you down to "bad" and 2 more would bring you down to crit?
I can see this being exploited by an antag or a shitter, because "it's a toy, not a weapon - I did nothing wrong!",

My main concerns are that majority of the station (including sec) dont have insuls, and the time between throws is incredibly quick - with no cooldown (like the nuke toy has - on its' interaction), this could be a really frustrating item for everyone that doesnt have it, or the means to defend themselves from it. It would likely be great against groups of people - and fantastic if 2 or more antags/shitters got their hands on these.

Not trying to hate lol - and this is probably my fav item in-game now (Once it gets added), but I think these are important things to consider, and wanted to know, myself.

Also just had another question about the object's deceleration; Does it keep going until it hits a wall, or slow down and eventually stop after a specific distance? I was unable to see that from the video lol.

**Comment by Cyber-Finn** [comment]
This is a really great addition to the game! GG!

**Comment by Cyber-Finn** [comment]
just had a few more questions on this, it's really interesting.

Are you able to hit with it in hand? (and does it electrocute?)
And does it shock players when you walk into them, with it in hand?

**Comment by TheShuEd** [comment]
no, this is not a weapon, and you cannot hit with it.  The throw's hit stun is only 0.8 seconds when the soap has 3 seconds, so it's much harder to use as a immobilization tool.  This PR does not add this toy to the maps, and I will leave the question of their number on the maps to Emisse and team

**Comment by Ilya246** [comment]
make emagged toy vendor dispense it maybe

**Comment by TheShuEd** [comment]
the toy singularity cannot be obtained in any way other than being in secret areas, and I’m thinking of leaving the same option for the toy Tesla

**Comment by TheShuEd** [comment]
why are 2 people already offering interaction with emag?  Is there something I don't know?  the toy singularity has nothing like it

**Comment by Cyber-Finn** [comment]
> the toy singularity cannot be obtained in any way other than being in secret areas, and I’m thinking of leaving the same option for the toy Tesla

THERES A TOY SINGULARITY????

**Comment by TheShuEd** [comment]
always was

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -822,6 +822,33 @@
...
+    requirePower: false
+  - type: EmitSoundOnCollide
+    sound:
+      path: /Audio/Effects/Lightning/lightningshock.ogg
+      params:
+        variation: 0.65
+        volume: -10
+    
+
 - type: entity
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:852]:
```suggestion
    
- type: entity
```

**Comment by K-Dynamic** [comment]
> why are 2 people already offering interaction with emag? Is there something I don't know? the toy singularity has nothing like it

it's a meme based on the fact emag have a lot of interaction with items already, often with nefarious or dangerous effects

**Pull request merged**